OpenSourcePolitics/decidim-app
# Backport/budgets projects searchable

#### :tophat: Description
Backport #12017 fix:projects displayed in search when not published

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #12017
- Related to #11721 
- [Notion card](https://www.notion.so/opensourcepolitics/Decidim-app-2-2-0-Unpublished-projects-displayed-in-search-32e8f353fb0041a8a9fe4d15be55fb38?pvs=4)

#### Testing
1. Duplicate a seed process (unpublished)
2. Go to new process components 
3. Create a budget
4. Create a project
5. In private browsing, see that the project is displayed in search for visitors, even though the process and the budget component are unpublished


#### Tasks

### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*
